### PR TITLE
New annotation object - WIP

### PR DESCRIPTION
**Pull request by drewlr**

### CODE REVIEW

File: remo/sdk.py [hunk 1]
```diff
@@ -336,10 +336,11 @@ def get_annotation_info(self, dataset_id: int, annotation_set_id: int, image_id:
         resp = self.api.get_annotation_info(dataset_id, annotation_set_id, image_id)
         return resp.get('annotation_info', [])
 
-    def get_annotation(self, dataset_id: int, annotation_set_id: int, image_id: int) -> Annotation:
+    def get_image_annotations(self, dataset_id: int, annotation_set_id: int, image_id: int) -> Annotation:
```
Comment (vmanilo) [inline remo/sdk.py]: @drewlr do we really need to rename this method?

File: remo/annotation_utils.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def parse_csv_obj_det(file_path) -> List[Annotation]:
         List[:class:`remo.Annotation`]
     """
 
-    annotations = []
+    annotations_list = []
```
Comment (vmanilo) [inline remo/annotation_utils.py]: @drewlr I don't see reason rename `annotations` to `annotations_list`

File: remo/sdk.py (hunk 1 above)
Comment (drewlr) [reply remo/sdk.py]: @vmanilo cuurent name is not very clear to me - I had no idea what it referred to. So unless we make it an hidden function, I'd say yes better to rename it

File: remo/annotation_utils.py (hunk 2 above)
Comment (drewlr) [reply remo/annotation_utils.py]: @vmanilo yes this we could leave "annotations"...I just find "annotation" and "annotations" very similar to each other, so I thought "annotations_list" makes it more obvious

File: remo/sdk.py (hunk 1 above)
Comment (vmanilo) [reply remo/sdk.py]: @drewlr what about `list_image_annotations`?

**Pull request merged**